templates/cloud/index.html

```diff
@@ -0,0 +1,251 @@
+{% extends "public-cloud/_base_public-cloud.html" %}
```

Comment:
Shouldn't this be `{% extends "public-cloud/_base-cloud.html" %}`
And do we still need the `public-cloud` folder or can we remove it?

Reply:
Good spot with the `{% extends "cloud/_base-cloud.html" %}`, thanks! And there has been no mention of removing /public-cloud yet. The _cloud_ and _overview_ are intended to both link to the same page. In the future this bubble may have more pages so there will be more use for the secondary nav.

Reply:
Ok it makes sense! Thanks for explaining!

Reply:
sorry, I don't think it should have a secondary nav

https://ubuntu.com/tco-calculator